reportportal-install.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "reportportal",
+    "description": "Send spec results to Report Portal",
+    "versions": [
+        {
+            "version": "1.0.1",
+            "gaugeVersionSupport": {
+                "minimum": "1.0.0",
+                "maximum": ""
+            },
+
+            "install": {
+                "windows": [],
+                "linux": [],
+                "darwin": []
+            },
+
+            "DownloadUrls": {
+                "x86": {
+                    "windows": "https://github.com/reportportal/agent-net-gauge/releases/download/1.0.1/reportportal-1.0.1-win-x86.zip",
+                    "linux": "",
```

Comment:
Although there isn't a x86 version of osx that I know of, there seem to be x86 versions of linux still being used. Would you consider publishing reportportal for x86 linux?

Reply:
Plugin is developed in `.net core`. Unfortunately it doesn't support linux x86. As soon as `.net core` supports it, plugin supports it as well. Hence this is not a mistake.

Reply:
fair enough, thank you for the explanation.